### PR TITLE
Bursar page body for v5

### PR DESCRIPTION
**Pull request by akeel230**
## Purpose
fix issue #216 

## Goals
Implement the bursar page according to the following design.
https://www.figma.com/file/3nKwsfWXeZIwb21Hee8ZFL/SACK-Site?node-id=411%3A3368

## Approach
Made the bursar body manually

### Screenshots
Screenshot (340)

  
##  Checklist
- [x] I have read and understood the development best practices guidelines
- [x] This PR doesn't commit any keys, passwords, tokens, usernames, or other secrets.
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation

## Related PRs
none

## Test environment
Jekyll , Live server

## Learning
Jekyll

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -6,3 +6,5 @@ git_source(:github) {|repo_name| "https://github.com/#{repo_name}" }
 
 # gem "rails"
 gem "jekyll"
+
+gem "webrick", "~> 1.7"
```
Comment (sathsarabandaraj) [inline Gemfile:10]: Why did you add 'webrick'. as I know webrick is also an HTTP server toolkit
We can use 'bundle exec jekyll server' to preview.
@kumuditha-udayanga please check this.

**Review by sathsarabandaraj — CHANGES_REQUESTED** [review]

**Review by sathsarabandaraj — APPROVED** [review]

File: Gemfile (hunk 1 above)
Comment (kumuditha-udayanga) [reply Gemfile:10]: Actually in some(including my self) without webrick "bundle exec jekyll serve" wont work, but what I usually do is i don't add the gem files to my PR

Comment (sathsarabandaraj) [reply Gemfile:10]: What if we add gem files to gitignore list.
We can use Dependabot to create PR if dependency update available.

**Comment by sathsarabandaraj** [comment]
@akeel230 can you add preview link.
https://docs.github.com/en/pages

File: bursar/index.html [hunk 2]
```diff
@@ -3,114 +3,122 @@
...
+          Sed ut perspiciatis unde omnis iste natus error sit voluptatem
+          accusantium doloremque laudantium, totam rem aperiam, eaque ipsa quae ab illo inventore
+          architecto beatae vitae dicta sunt explicabo. Nemo enim ipsam voluptatem quia voluptas sit
+          aspernatur aut odit aut fugit?
+          Sed ut perspiciatis unde omnis iste natus error sit voluptatem
+          accusantium doloremque laudantium, totam rem aperiam, eaque ipsa quae ab illo inventore
+          architecto beatae vitae dicta sunt explicabo. Nemo enim ipsam voluptatem quia voluptas sit
+          aspernatur aut odit aut fugit?
+        </p>
+        <h2 class="mt-0 text-center"><button type="button" class="btn btn-info text-center font-weight-bold">See
```
Comment (kumuditha-udayanga) [inline bursar/index.html:109]: Is this <h2> necessary?

File: bursar/index.html [hunk 3]
```diff
@@ -3,114 +3,122 @@
...
-        </div>
-        <div class="section pb-0" id="sds-fee">
-          <div class="container text-left">
-            <h2 class="mt-0">SDS Fee</h2>
-            <p class="description pb-5">
-              Sed ut perspiciatis unde omnis iste natus error sit voluptatem
-              accusantium doloremque laudantium, totam rem aperiam, eaque ipsa quae ab illo inventore
-              architecto beatae vitae dicta sunt explicabo. Nemo enim ipsam voluptatem quia voluptas sit
-              aspernatur aut odit aut fugit?
+            <p>
```
Comment (kumuditha-udayanga) [inline bursar/index.html:63]: Remove the <p> tag opening and closing ones

**Comment by kumuditha-udayanga** [comment]
Thank you @akeel230, awesome work!.

**Pull request merged**